ui/page/tickets/activity_page.go

```diff
@@ -103,6 +105,13 @@ func (pg *ActivityPage) Layout(gtx layout.Context) layout.Dimensions {
 							})
 						})
 					}),
+					layout.Expanded(func(gtx C) D {
+						if pg.orderDropDown.IsOpen() || pg.ticketTypeDropDown.IsOpen() || pg.walletDropDown.IsOpen() {
+							return pg.backdrop.Layout(gtx)
```

Comment:
This implementation requires each page to have a backdrop, the backdrop should be handled by the dropdown instead.